Shared/sdk/net/Packets.h

```diff
@@ -149,6 +149,5 @@ enum ePacketID
     PACKET_ID_PLAYER_ACINFO,
     PACKET_ID_CHAT_CLEAR,
     PACKET_ID_SERVER_INFO_SYNC,
-    PACKET_ID_DISCORD_JOIN,
     PACKET_ID_PLAYER_RESOURCE_START,
```

Comment:
I think this causes desync problem.  Will we merge this in 1.6?

Reply:
We could keep the packet declared here but not use it, I think that's fine